chia/plotting/check_plots.py

```diff
@@ -153,35 +164,52 @@ def check_plots(
                             if proof_spent_time > 15000:
                                 log.warning(
                                     f"\tFinding proof took: {proof_spent_time} ms. This should be below 15 seconds "
-                                    f"to minimize risk of losing rewards."
+                                    f"to minimize risk of losing rewards. Filepath: {plot_path}"
                                 )
                             else:
-                                log.info(f"\tFinding proof took: {proof_spent_time} ms")
-                            total_proofs += 1
+                                log.info(f"\tFinding proof took: {proof_spent_time} ms. Filepath: {plot_path}")
+
                             ver_quality_str = v.validate_proof(pr.get_id(), pr.get_size(), challenge, proof)
                             assert quality_str == ver_quality_str
                         except AssertionError as e:
-                            log.error(f"{type(e)}: {e} error in proving/verifying for plot {plot_path}")
+                            log.error(
+                                f"{type(e)}: {e} error in proving/verifying for plot {plot_path}. Filepath: {plot_path}"
+                            )
                             caught_exception = True
                         quality_start_time = int(round(time() * 1000))
                 except KeyboardInterrupt:
                     log.warning("Interrupted, closing")
-                    return None
+                    return
                 except SystemExit:
                     log.warning("System is shutting down.")
-                    return None
+                    return
                 except Exception as e:
                     log.error(f"{type(e)}: {e} error in getting challenge qualities for plot {plot_path}")
                     caught_exception = True
-                if caught_exception is True:
-                    break
```

Comment:
removing this means we'll continue checking more plots, after logging an error. note that we still treat it as a bad plot below, since `caught_exception` is `True`

Reply:
I will restore this. talking to Harold, the expected case where we're currently throwing should probably not throw in the final version